http-clients/netty-nio-client/src/test/java/software/amazon/awssdk/http/nio/netty/internal/OldConnectionReaperHandlerTest.java

```diff
@@ -0,0 +1,49 @@
+package software.amazon.awssdk.http.nio.netty.internal;
```

Comment:
Hmm, seems missing copyright headers on the test classes. (We need to add the checkstyles for test files)

Reply:
Fixed